source/common/runtime/runtime_features.cc

```diff
@@ -45,6 +45,41 @@ constexpr const char* disallowed_features[] = {
     // Acts as both a test entry for deprecated.proto and a marker for the Envoy
     // deprecation scripts.
     "envoy.deprecated_features.deprecated.proto:is_deprecated_fatal",
+    // 1.12.0
+    "envoy.deprecated_features.route.proto:regex_match",
+    "envoy.deprecated_features.route.proto:regex",
+    "envoy.deprecated_features.accesslog.proto:config",
```

Comment:
We just did this 1-2 weeks ago so this seems pretty mean for those running on master. Defer to next release? Although aren't we moving to a 6 month cycle for this anyway? I can't remember. (Applies to all config in this list). Also I think this deprecation isn't even fully completed yet as @lizan identified some stragglers. cc @yanavlasov

Reply:
Do we want to leave everything with config until the next release then?  I don't think we can do the stragglers in this release

Reply:
+1 let's just leave until next release.